app/client/packages/mcp/.env.example

```
@@ -0,0 +1,73 @@
+# shipped default so copying it does not silently enable the privileged tool layers.
+#
+# Note what APPSMITH_MCP_ENABLED does NOT do here: standalone, `start-server.sh` execs the bundle unconditionally
+# and `server.ts` reads only the DATA and JS gates below — the top-level gate is enforced by run-mcp.sh, Caddy, and
+# SecurityConfig, i.e. only inside the container. So on this path it is a documentation marker; the two gates that
+# actually change standalone behaviour are the DATA and JS ones.
+#
+# For a local dev loop against the data/JS tools (including live-harness.mjs, which now exits non-zero on a failed
+# check), set APPSMITH_MCP_DATA_ENABLED=1 and APPSMITH_MCP_JS_ENABLED=1 below.
+APPSMITH_MCP_ENABLED=0
+APPSMITH_MCP_PORT=8092
+APPSMITH_API_BASE_URL=http://127.0.0.1:8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPSMITH_API_BASE_URL key should go before the APPSMITH_MCP_ENABLED key

(UnorderedKey)